docs/en/sql-reference/aggregate-functions/reference/argmin.md

```diff
@@ -49,3 +50,65 @@ Result:
 │ worker               │
 └──────────────────────┘
 ```
+
+**Extended example**
+
+```sql
+CREATE TABLE test
+(
+    a Nullable(String),
+    b Nullable(Int64)
+)
+ENGINE = Memory AS
+SELECT *
+FROM values((NULL, 0), ('a', 1), ('b', 2), ('c', 2), (NULL, NULL), ('d', NULL));
+
+select * from test;
+┌─a────┬────b─┐
+│ ᴺᵁᴸᴸ │    0 │
+│ a    │    1 │
+│ b    │    2 │
+│ c    │    2 │
+│ ᴺᵁᴸᴸ │ ᴺᵁᴸᴸ │
+│ d    │ ᴺᵁᴸᴸ │
+└──────┴──────┘
+
+select argMin(a, b), min(b) from test;
+┌─argMin(a, b)─┬─min(b)─┐
+│ a            │      0 │ -- argMin = a because it the first not-Null value, min(b) is from another row!
+└──────────────┴────────┘
+
+select argMin(tuple(a), b) from test;
+┌─argMin(tuple(a), b)─┐
+│ (NULL)              │ -- Tuple allows to get Null value.
+└─────────────────────┘
+
+select (argMin((a, b), b) as t).1 argMinA, t.2 argMinB from test;
+┌─argMinA─┬─argMinB─┐
+│ ᴺᵁᴸᴸ    │       0 │ -- you can use Tuple and get both (all - tuple(*)) columns for the according max(b)
+└─────────┴─────────┘
+
+select argMin(a, b), min(b) from test where a is Null and b is Null;
+┌─argMin(a, b)─┬─min(b)─┐
+│ ᴺᵁᴸᴸ         │   ᴺᵁᴸᴸ │ -- Nulls are not skipped because only Null values are available
```

Comment:
```suggestion
│ ᴺᵁᴸᴸ         │   ᴺᵁᴸᴸ │ -- ll aggregated rows contains at least one `NULL` value because of the filter, so all rows are skipped, therefore the result will be `NULL`
```